### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by pjurasek**
Typo update

### CODE REVIEW

File: Structural/Facade/BiosInterface.php [hunk 1]
```diff
@@ -3,29 +3,29 @@
...
 
 /**
- * Class BiosInterface.
+ * Interface BiosInterface
  */
 interface BiosInterface
 {
     /**
-     * execute the BIOS.
+     * execute the BIOS
```
Comment (bocharsky-bw) [inline Structural/Facade/BiosInterface.php]: I think we should keep dot at the ending of description

Comment (rob006) [inline Structural/Facade/BiosInterface.php]: If we consider description as a sentence, it should be capitalized.

Comment (bocharsky-bw) [inline Structural/Facade/BiosInterface.php]: :+1: for capitalized

**Comment by domnikl** [comment]
I would be happy to merge this if you capitalized the comments and re-added the dots @pjurasek!

**Comment by pjurasek** [comment]
I capitalized the comments and re-added the dots @domnikl

**Comment by domnikl** [comment]
Merged, thanks!

**Pull request merged**